spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -102,6 +102,14 @@ module MiqAeServiceSpec
       it "expose all expected active_record models as service_models" do
         excluded_model_names = %w(
           AuthToken
+          AuthenticationAllowAll
+          AuthenticationGithub
+          AuthenticationGoogle
+          AuthenticationHtpasswd
+          AuthenticationLdap
+          AuthenticationOpenId
+          AuthenticationRequestHeader
+          AuthenticationRhsm
```

Comment:
@alongoldboim can you look over this list.  I bet that this list was updated before you added the service models from [this discussion](https://github.com/ManageIQ/manageiq/pull/7517/files#r73168540).